helm/superset/templates/_helpers.tpl

```diff
@@ -89,6 +89,21 @@ WTF_CSRF_ENABLED = True
 WTF_CSRF_EXEMPT_LIST = []
 # A CSRF token that expires in 1 year
 WTF_CSRF_TIME_LIMIT = 60 * 60 * 24 * 365
+{{- if .Values.supersetNode.connections.redis_password }}
+class CeleryConfig(object):
+  BROKER_URL = f"redis://{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
```

Comment:
Are you sure that it gonna works? Celery [documentation use](https://docs.celeryproject.org/en/stable/getting-started/backends-and-brokers/redis.html#configuration) `redis://:password@hostname:port/db_number`, nor `redis://password@hostname:port/db_number` (compare collon before password).

Reply:
Hmm... that's weird, I tested it in minikube and it worked. I've checked logs in both superset and redis (no errors) and info about successfully connected clients in redis. 
```
DB 0: 3 keys (0 volatile) in 4 slots HT.
10 clients connected (0 replicas), 1022392 bytes in use
```
But docs says different thing and you've got right that it's better to stick to documentation.